core/internal/gethwrappers/generated/flux_aggregator_wrapper/flux_aggregator_wrapper.go

```diff
@@ -370,9 +370,9 @@ func (_FluxAggregator *FluxAggregatorCallerSession) GetOracles() ([]common.Addre
 	return _FluxAggregator.Contract.GetOracles(&_FluxAggregator.CallOpts)
 }
 
-// GetRoundData is a free data retrieval call binding the contract method 0x9a6fc8f5.
+// GetRoundData is a free data retrieval call binding the contract method 0x0720da52.
 //
-// Solidity: function getRoundData(uint80 _roundId) view returns(uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound)
```

Comment:
These should stay as uint80. I think rebasing and running `make go-solidity-wrappers` should fix this.

Reply:
Yeah not really sure how this happened.

Reply:
Actually, nvm.